docs/reference/query-dsl/geo-polygon-query.asciidoc

```diff
@@ -6,25 +6,29 @@ points. Here is an example:
 
 [source,js]
 --------------------------------------------------
+GET /_search
 {
-    "bool" : {
-        "query" : {
-            "match_all" : {}
-        },
-        "filter" : {
-            "geo_polygon" : {
-                "person.location" : {
-                    "points" : [
+    "query": {
+        "bool" : {
+            "must" : {
```

Comment:
It is only cacheable if it is inside the `bool.filter` clause, so I would leave it there.  (although perhaps delete the `bool.must.match_all`?